guides/v2.2/contributor-guide/templates/basic_template.md

```diff
@@ -128,20 +128,18 @@ Code blocks can also be defined using [Rouge formatting](http://rouge.jneen.net/
 
 For inline code, surround the content with single backticks: `` `example` ``.
 
-For blocks of code, surround content with 3 backticks and an optional [supported language](https://GitHub.com/jneen/rouge/wiki/List-of-supported-languages-and-lexers){:target="_blank"}.
+For blocks of code, surround content with 3 backticks and a [supported language](https://GitHub.com/jneen/rouge/wiki/List-of-supported-languages-and-lexers){:target="_blank"}.
 
 *Example:*
 
-```markdown
-```html
+```text
```

Comment:
The example should include backticks and should look like:
````
```html
<div class="container">
  <h4 class="title">Title</h4>
  <div class="content">
    <p>Paragraph content.</p>
  </div>
</div>
```
````
See [present view]((https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/basic_template.html#code-blocks)).